components/higher-order/with-focus-return/test/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import { Component } from '../../../../element';
+
+/**
+ * Internal dependencies
+ */
+import withFocusReturn from '../';
+
+class Test extends Component {
+	render() {
+		return (
+			<div className="test">Testing</div>
+		);
+	}
+}
+
+describe( 'withFocusReturn()', () => {
+	describe( 'expected behavior', () => {
```

Comment:
Do we need this second `describe` block? Our tests are all testing "expected behavior" after all.